js/forum/src/addSubscriptionSettings.js

```diff
@@ -13,5 +13,13 @@ export default function() {
         onchange: this.preferenceSaver('followAfterReply')
       })
     );
+
+    items.add('followNewDiscussions',
+      Switch.component({
+        children: 'Follow new discussions automatically',
```

Comment:
Please add a translation key here. If you have the time, please also send a PR to flarum/flarum-ext-english. (If you don't, that's fine, then we'll add it.)